app/Commands/VisitCommand.php

```diff
@@ -128,9 +129,13 @@ protected function makeRequest(): array
             $request->withoutRedirecting();
         }
 
-        $response = $method === 'get'
-            ? $request->$method($url)
-            : $request->$method($url, $this->getPayload());
+        try {
+            $response = $method === 'get'
+                ? $request->$method($url)
+                : $request->$method($url, $this->getPayload());
+        } catch (ConnectionException) {
```

Comment:
I don't think each connection exception will be caused by an invalid url. Create a new renderable `CouldNotMakeRequest` exception and pass it the message from `ConnectionException`.